### ISSUE TITLE
Update Dependencies

### ISSUE THREAD

**Issue by Dargmuesli** [mikeerickson/gulp-phplint#42]
Hey, installing `gulp-phplint` currently gives the following <del>errors</del> warnings:

```Bash
warning gulp-phplint > gulp-messenger > gulp > vinyl-fs > graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
warning gulp-phplint > gulp-messenger > gulp > vinyl-fs > glob-stream > minimatch@2.0.10: Please update to minimatch 3.0.2 orhigher to avoid a RegExp DoS issue
warning gulp-phplint > gulp-messenger > gulp > vinyl-fs > glob-stream > glob > minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning gulp-phplint > gulp-notify > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
warning gulp-phplint > gulp-messenger > gulp > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
warning gulp-phplint > gulp-messenger > lodash.isarray@4.0.0: This package is deprecated. Use Array.isArray.
warning gulp-phplint > gulp-messenger > gulp > vinyl-fs > glob-watcher > gaze > globule > minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning gulp-phplint > gulp-messenger > gulp > vinyl-fs > glob-watcher > gaze > globule > glob > minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
warning gulp-phplint > gulp-messenger > gulp > vinyl-fs > glob-watcher > gaze > globule > glob > graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
```

Seems like the dependencies need some updates!

**Comment by mikeerickson** [comment]
@Dargmuesli They are not errors per se (all warnings, well known warnings in fact).  But they should be cleared nonetheless.   It should not affect use of the plugin.  I will get this added to my todo list :-)

**Comment by Dargmuesli** [comment]
Correct. Most of them originate from gulp-messenger though. Thanks for looking at this!

Just for your convenience I'll hand you a PR in that updates all dependencies to the most recent version. Feel free to just close it if anything breakes :)

**Comment by mikeerickson** [comment]
Ya, I was not prepared to jump into the updates for `gulp-messenger` today (admit that this has lower priority these days based on gulp popularity)  But it is something that needs spring cleaning :-)

**Comment by mikeerickson** [comment]
All these issues have been resolved with the 0.8.0 release

```
npm install gulp-phplint@0.8.0
```
